ci-operator/templates/openshift/installer/cluster-launch-installer-ovirt-e2e.yaml

```diff
@@ -417,6 +423,7 @@ objects:
 
         # What we're doing here is we generate manifests first and force that OpenShift SDN is configured.
         TF_LOG=debug openshift-install --dir=/tmp/artifacts/installer create manifests --log-level=debug
+        cat /tmp/artifacts/installer/manifests/cluster-infrastructure-02-config.yml
```

Comment:
Is this needed?

Reply:
That more for debugging purposes of CI failures. 
@Gal-Zaidman please leave a comment above this.

Reply:
Added A comment

Reply:
This doesn't leak any form of long term credentials does it?

Reply:
It doesn't appear to from the rehearsal logs.